src/app/layers/layer.model.ts

```diff
@@ -39,9 +39,9 @@ export class VectorData extends LayerData<Array<ol.Feature>> {
     static olParse(time: Time,
                    projection: Projection,
                    extent: [number, number, number, number],
-                   source: (Document | Node | GlobalObject | string),
+                   source: (Document | Node | any | string),
                    opt_options?: olx.format.ReadOptions): VectorData {
-        return new VectorData(time, projection, new ol.format.GeoJSON().readFeatures(source, opt_options), extent);
+        return new VectorData(time, projection, new OlFormatGeoJSON().readFeatures(source, opt_options), extent);
     }
 
     constructor(time: Time, projection: Projection, data: Array<ol.Feature>, extent: [number, number, number, number]) {
```

Comment:
Why does `ol.Feature` work if you changed the above from `ol.format.GeoJson` to `OlFormatGeoJSON`?

Reply:
I guess this is a "feature" of the typings.
The documentation for the module (https://www.npmjs.com/package/ol) states that you have to import everything explicitly.